src/main/kotlin/no/nav/amt/deltaker/bff/deltaker/api/model/DeltakerResponse.kt

```diff
@@ -73,6 +75,17 @@ data class DeltakerResponse(
         val ledetekst: String?,
         val innhold: List<Innholdselement>,
     )
+
+    data class DeltakelsesmengderDto(
+        val nesteDeltakelsesmengde: DeltakelsesmengdeDto?,
```

Comment:
det er ikke helt åpenbart for meg hva vi mener med "neste" og "siste". Er "siste" den som gjelder nå, mens "neste" er den som ligger klar med en gyldigFra frem i tid?

Reply:
Jeg tror det blir vanskelig å finne selvforklarende navn for disse (særlig "siste") uansett 😅 Som du sier er det nok lettere når man ser dem i bruk.